ilc/server/tailor/configs-injector.spec.js

```diff
@@ -441,6 +484,7 @@ describe('configs injector', () => {
                             `<link rel="alternate" hreflang="en-us" href="https://test.com/test/route" data-ilc="1">` +
                             `<link rel="alternate" hreflang="ua-ua" href="https://test.com/ua/test/route" data-ilc="1">` +
                             `<link rel="alternate" hreflang="x-default" href="https://test.com/test/route" data-ilc="1">` +
+                            `<link rel="canonical" href="https://test.com/test/route" data-ilc="1" />` +
```

Comment:
we need to ensure that canonical always ends on `/`

Reply:
btw the same is probably applicable to the `hreflang` as well, because we now redirect customers from the URLs that are not ending on `/` the path that ends on `/` in order to protect from duplicating URLs in search engines. So I think it worth to generate both canonicals and hreflang with trailing slash